### PR TITLE
(Publish 26 June 2024 at 9:00 AM) add browser logs API docs

### PR DESCRIPTION
**Pull request by metal-messiah**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context
This feature will go out with Browser Agent Version 1.261.0, set to release next week.  This is adding API document for that new feature.

https://github.com/newrelic/newrelic-browser-agent/pull/1051

### CODE REVIEW

**Review by patrickhousley — APPROVED** [review]

File: src/content/docs/browser/new-relic-browser/browser-apis/log.mdx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+      Logging
+    ]
+  }
+  ```
+
+  For more information, see the [npm browser installation documentation](https://www.npmjs.com/package/@newrelic/browser-agent#new-relic-browser-agent).
+
+## Description
+
+Upon executing this function with a valid message and elective options, the browser agent records the data as a single `log` event.  See the [Logs UI](https://docs.newrelic.com/docs/logs/ui-data/use-logs-ui/) for more information about log events. Any custom attributes supplied to the API call in the `options` argument (`options.customAttributes`) will be appended as top-level attributes on the log event. You can control the `logType` of the captured log by supplying a `level` to the `options` argument (`options.level`), which defaults to `info`.
```
Comment (rhetoric101) [inline src/content/docs/browser/new-relic-browser/browser-apis/log.mdx:44]: @metal-messiah this sentence makes it seem like the browser is the one executing the function. Can we be more specific and say something like this:

When you execute this function with a valid message and elective options, the browser agent records the data as a single `log` event.

File: src/content/docs/browser/new-relic-browser/browser-apis/log.mdx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    </tr>
+    <tr>
+      <td>
+        `options`
+
+        _Object_
+      </td>
+
+      <td>
+        Optional. An object used for supplying optional configurations for the captured log. `options.customAttributes` is an object of key:val pairs that assigns a top-level property and value to the created log for each attribute supplied.  `options.level` is an enum that assigns a log level to the created log event.  The `level` must be one of: `debug | error | info | trace | warn`.  The log level defaults to `info` if not supplied.
```
Comment (rhetoric101) [inline src/content/docs/browser/new-relic-browser/browser-apis/log.mdx:81]: Could we rephrase this?

CURRENT: `options.level` is an enum that assigns a log level to the created log event.
PROPOSED:  The enum `options.level` assigns a log level to the created log event.

File: src/content/docs/browser/new-relic-browser/browser-apis/using-browser-apis.mdx [hunk 3]
```diff
@@ -137,7 +157,7 @@ To report errors in your application, use these methods:
 
     <tr>
       <td>
-        Log a caught or handled error, without disrupting your app's operation
+        Capture a caught or handled error, without disrupting your app's operation
```
Comment (rhetoric101) [inline src/content/docs/browser/new-relic-browser/browser-apis/using-browser-apis.mdx:160]:
```suggestion
        Capture a caught or handled error without disrupting your app's operation
```

File: src/content/docs/browser/new-relic-browser/browser-apis/wraplogger.mdx [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+    features: [
+      Logging
+    ]
+  }
+  ```
+
+  For more information, see the [npm browser installation documentation](https://www.npmjs.com/package/@newrelic/browser-agent#new-relic-browser-agent).
+
+## Description
+Upon providing this method with a valid parent container and child function name, the browser agent will record a new `log` event every time the wrapped function is `invoked`, capturing the `first` argument passed to the invoked function as the log's `message`. See the [Logs UI](https://docs.newrelic.com/docs/logs/ui-data/use-logs-ui/) for more information about log events. Optional configurations can be passed along with these captured logs, by use of the `options` argument. Any custom attributes supplied to the API call in the `options` argument (`options.customAttributes`) will be appended as top-level attributes on every log event created by this wrapper. You can control the `logType` of the captured log by supplying a `level` to the `options` argument (`options.level`), which defaults to `info`. **Note** once successfully wrapped, the function's logging detection will not be able to be altered.
```
Comment (rhetoric101) [inline src/content/docs/browser/new-relic-browser/browser-apis/wraplogger.mdx:43]: @metal-messiah this is another case where it almost reads like it's the browser who's providing the container. Also, could we divide this long sentence into two? Is the proposed revision below accurate?

Comment (rhetoric101) [inline src/content/docs/browser/new-relic-browser/browser-apis/wraplogger.mdx:43]:
```suggestion
After you provide this method with a valid parent container and child function name, the browser agent will record a new log event every time the wrapped function is invoked. The first argument is passed to the invoked function as the log's message. See the [Logs UI](https://docs.newrelic.com/docs/logs/ui-data/use-logs-ui/) for more information about log events. 

Optional configurations can be passed along with these captured logs with the `options` argument. Any custom attributes supplied to the API call in the `options` argument (`options.customAttributes`) will be appended as top-level attributes on every log event created by this wrapper. You can control the `logType` of the captured log by supplying a `level` to the `options` argument (`options.level`), which defaults to `info`. Note that once successfully wrapped, the function's logging detection can't be altered.
```

**Review by rhetoric101 — COMMENTED** [review]
@metal-messiah thanks for these new API additions! I just inserted a few minor wording suggestions. See what you think.

**Review by rhetoric101 — APPROVED** [review]
@metal-messiah thanks for making these changes. These look good to me!

**Pull request merged**